arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-nigeria.dts

```diff
@@ -386,9 +386,10 @@
 &i3c4 {
 	status = "okay";
 	initial-role = "primary";
-	bus-context = /bits/ 8 <I3C_BUS_CONTEXT_JESD403>;
-	i3c-scl-hz = <6000000>;
+	internal-pullup = <2>;
```

Comment:
Commit message update is required based on the changes you added here.

Reply:
> Looks like missing bus-context tag removal in the commit message.

Done.